tox.ini

```
@@ -2,12 +2,12 @@
 min_version =  4.11
 
 env_list =
-    python{3.8,3.9,3.10,3.11}-django4.2-wagtail{5.2,6.0,6.1}
-    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0,6.1}
+    python{3.9,3.10,3.11}-django4.2-wagtail{5.2,6.0,6.1,6.2,6.3}
+    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0,6.1,6.2,6.3}
+    python{3.10,3.11,3.12}-django5.1-wagtail6.3
```

```suggestion
    python{3.9,3.10,3.11}-django4.2-wagtail{5.2,6.2,6.3}
    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.2,6.3}
    python{3.10,3.11,3.12}-django5.1-wagtail6.3
```

since 6.0 and 6.1 are no longer officially supported